docs/launch/local-db.md

```diff
@@ -356,11 +372,13 @@ return (
 );
 ```
 
+<!--Ry - I believe we said that we'd make this tutorial about xmtp-js and the code is not about React. This code below refers to a react hook - but it is a Dexie react hook that you can use with xmtp-js - and not an XMTP React SDK hook?-->
+
```

Comment:
A question for Ry above in HTML comment format.....

Reply:
addressed in the first comment asking this question...